XLS tests are locale-dependent

The tests for XLS importer can fail depending on the locale they are run with.

### To Reproduce
Steps to reproduce the behavior: run `mvn test` in the `fr.FR` locale (unfortunately there does not seem to be a CLI argument to set that).

### Current Results

```
[ERROR] readXlsxAsText(com.google.refine.importers.ExcelImporterTests)  Time elapsed: 0.1 s  <<< FAILURE!
java.lang.AssertionError: expected [1.1] but found [1,1]
	at com.google.refine.importers.ExcelImporterTests.readXlsxAsText(ExcelImporterTests.java:224)
```

### Expected Behavior
The tests should pass.

### Versions<!--  (please complete the following information)-->
 - OpenRefine: master branch


### Additional context
Intuitively the locale dependency of OpenRefine's behaviour is expected here, since we are just preserving the string representation of values in the XLS file, which are locale dependent, so I would say we should just ensure the tests accept that.

It could be even better to give the user the option to select the locale in which the values should be represented, but I am not even sure POI lets us configure that.

--- a/main/tests/server/src/com/google/refine/importers/ExcelImporterTests.java
+++ b/main/tests/server/src/com/google/refine/importers/ExcelImporterTests.java
@@ -44,6 +44,8 @@ import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
+import java.text.DecimalFormat;
+import java.text.NumberFormat;
 import java.time.LocalDate;
 import java.time.Month;
 import java.time.OffsetDateTime;
@@ -85,6 +87,7 @@ public class ExcelImporterTests extends ImporterTest {
 
     private static final File xlsFileWithMultiSheets = createSheetsWithDifferentColumns(false);
     private static final File xlsxFileWithMultiSheets = createSheetsWithDifferentColumns(true);
+    private static final NumberFormat NUMBER_FORMAT = DecimalFormat.getInstance();
 
     @Override
     @BeforeTest
@@ -221,8 +224,8 @@ public class ExcelImporterTests extends ImporterTest {
 
         Assert.assertEquals(project.rows.size(), ROWS);
         Assert.assertEquals(project.rows.get(1).cells.size(), COLUMNS);
-        Assert.assertEquals(((String) project.rows.get(1).getCellValue(0)), "1.1");
-        Assert.assertEquals(((String) project.rows.get(2).getCellValue(0)), "2.2");
+        Assert.assertEquals(((String) project.rows.get(1).getCellValue(0)), NUMBER_FORMAT.format(1.1));
+        Assert.assertEquals(((String) project.rows.get(2).getCellValue(0)), NUMBER_FORMAT.format(2.2));
 
         Assert.assertEquals((String) project.rows.get(1).getCellValue(1), "FALSE");
         Assert.assertEquals((String) project.rows.get(2).getCellValue(1), "TRUE");

